superset-frontend/src/explore/components/DataTablesPane.tsx

```diff
@@ -73,16 +73,21 @@ export const DataTablesPane = ({
   queryFormData,
   tableSectionHeight,
   onCollapseChange,
+  chartStatus,
 }: {
   queryFormData: Record<string, any>;
   tableSectionHeight: number;
   onCollapseChange: (openPanelName: string) => void;
+  chartStatus: string;
```

Comment:
Can this be string literals or some shared typing as I assume there are only a limited set of state?